static/js/util/useNotificationHelper.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+import {
+  Notification,
+  NotificationHelper,
+  QueuedNotification,
+} from "types/notification";
+import isEqual from "lodash/isEqual";
+import { useLocation } from "react-router-dom";
+
+const useNotificationHelper = (): NotificationHelper => {
+  const location = useLocation() as QueuedNotification;
+  const [notification, setNotification] = useState<Notification | null>(
+    location.state?.queuedNotification ?? null
+  );
+
+  window.history.replaceState({}, "");
```

Comment:
Do we need this on every re-render?

Reply:
This is a leftover from the previous implementation wit sharednotufy context. I'll clean this up.